keras/preprocessing/text.py

```diff
@@ -154,11 +154,11 @@ def hashing_trick(
     Args:
         text: Input text (string).
         n: Dimension of the hashing space.
-        hash_function: defaults to python `hash` function, can be 'md5' or
+        hash_function: when None uses a python `hash` function, can be 'md5' or
```

Comment:
Please add a comma after `None` and use backticks. Also a period before "can"

Reply:
Done